pipeline/scripts/download-electron-mirror.js

```diff
@@ -7,7 +7,7 @@ const fs = require('fs');
 
 download(
     {
-        version: pkg.devDependencies.electron,
+        version: pkg.dependencies.electron,
```

Comment:
This is unrelated - we recently moved electron to dependencies from devDependencies and missed this script.